api/v1/awsloadbalancercontroller_types.go

```diff
@@ -116,11 +116,19 @@ type AWSLoadBalancerControllerSpec struct {
 	// The secret is required to be in the operator namespace.
 	// If this field is empty, the credentials will be
 	// requested using the Cloud Credentials API,
-	// see https://docs.openshift.com/container-platform/4.11/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html.
+	// see https://docs.openshift.com/container-platform/4.13/authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.html.
 	//
 	// +kubebuilder:validation:Optional
 	// +optional
 	Credentials *configv1.SecretNameReference `json:"credentials,omitempty"`
+
+	// credentialsRequestConfig specifies further customization options for the controller's CredentialsRequest.
+	// This field won't have any effect if credentials have already been provided through the `Credentials` field,
+	// as a request for credentials from the Cloud Credentials Operator will not be triggered.
+	//
+	// +kubebuilder:validation:Optional
```

Comment:
You don't need this, `+optional` is sufficient

Reply:
Thanks for the hint, I just wanted to keep it consistent with the rest of the API. 

However I recalled some discussions around the usage of `//optional` vs kubebuilder's optional. I believe the conclusion was that since it's unlikely we'll use anything except from kubebuilder to do code generation it may be more pragmatical to keep only kubebuidler's one.